staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -1006,6 +1006,84 @@ func (f *fakeTimeBudget) takeAvailable() time.Duration {
 
 func (f *fakeTimeBudget) returnUnused(_ time.Duration) {}
 
+func TestStartingResourceVersion(t *testing.T) {
```

Comment:
nit: Moving this function makes for a diff that is difficult to review.

Reply:
Yes - but in order to fake dispatchBudget (which is private field) it has to be here :(